lib/cdhc/enormp.c

```diff
@@ -49,7 +49,7 @@ double Cdhc_enormp(double x)
     }
     else {
         x2 = x * x;
-        x4 *= 1.;               /* huh, what? See original FORTRAN */
+        // x4 *= 1.;               /* huh, what? See original FORTRAN */
```

Comment:
```suggestion
        /* x4 *= 1.; */               /* huh, what? See original FORTRAN */
```

just a minor: AFAIK C++ style comments are undesired in GRASS (maybe only in the past but probably what `indent -sc ...`does in https://github.com/OSGeo/grass/blob/main/utils/grass_indent.sh)

Reply:
I don't know about connection to `indent -sc` that seems to be about adding stars at the line beginning for multi-line comments, but there is no need for the "no C++ comments" rule anymore. C99 which is our minimum has `// comment`. See https://en.cppreference.com/w/c/comment

There is also no need to keep useless code around (if that's indeed the case here).

Reply:
Even if C99+ allows // comments in C code (and nowadays without compiler problem), we as a project may have style guides that discourage it. (GNU indent has no problem with them).
I personally prefer // comments (at least for trailing comments), but I think this is not the place for that discussion. We will probably return to it in connection with #2272.

The "huh, what? See original FORTRAN" comment was helpful for starting research on this code (even if it ended up almost empty handed). What about deleting the line:

```c
x4 *= 1.;               /* huh, what? See original FORTRAN */
```


and adding a comment to the "living" code:
```c
x4 = 0.5;               /* was 'x4=1.0D0*x4' in original Fortran code */
```

?

Reply:
`// comments`: As far as I know, the reason there was a guideline against these was that some compilers did not support them.

`x4 = 0.5;...was 'x4=1.0D0*x4`: The reasoning in the suggested comment is not obvious to me.

Reply:
Suggestion?

Reply:
The problem is that there is no trace of who changed to "x4 = 0.5;"  and when. That is pre-version control code. Probably by the original writer.

Reply:
Then I think I missed something like "...but was changed some time later."

Reply:
`x4 = x * x;` assuming that there is an error in the original Fortran code and `x4 = x*x` should be before, not after `IF (x1.GT.0.5D0) GO TO 10`.

Alternatively, given that `Cdhc_normp()` calculates normal distribution probabilities, one could wildly guess that `Cdhc_enormp()` calculates normalized (mean = 0) exponential distribution probabilities and have a look at [gsl](https://www.gnu.org/software/gsl/doc/html/index.html) for inspiration what this function should look like.

Reply:
Another solution would be to remove `Cdhc_enormp()` because 1) this function is not used, 2) there is no description what it is supposed to do, thus useless.

Reply:
> Another solution would be to remove `Cdhc_enormp()` because 1) this function is not used, 2) there is no description what it is supposed to do, thus useless.

Just what I have been thinking! There are tests using it in the Fortran code, but not used here.

Reply:
+1 @metzm Removal is probably safe even considering custom user code given it is not properly documented (it is listed the doc). ...but in a separate PR.

Reply:
Good! I'll lift this change from this PR and open another one for the file removal.